Incorrect loglevel set in  cherrypy/cherrypy/process/win32.py  file

Its looks like a trival issue. But want to inform you.

The loglevel set for class ConsoleCtrlHandler(plugins.SimplePlugin) is error for success case which I assume should be info or debug.

That is in class ConsoleCtrlHandler(plugins.SimplePlugin), start and stop method we have the below logs which are set to error.
start: 
 self.bus.log('Set handler for console events.', level=40) 
stop:
 self.bus.log('Removed handler for console events.', level=40)

I assume that the log level should be 10 or 20 if I'm not wrong.

**I'm submitting a ...**
- [ ] bug report


**Do you want to request a *feature* or report a *bug*?**
Reporting as bug


**What is the current behavior?**
Currently it logs as ERROR as below
 [cherrypy.error] [ERROR] [18/Dec/2018:19:40:27] ENGINE Set handler for console events.


**If the current behavior is a bug, please provide the steps to reproduce and if possible a screenshots and logs of the problem. If you can, show us your code.**
File name:  cherrypy/cherrypy/process/win32.py
Class: ConsoleCtrlHandler(plugins.SimplePlugin)
Method:
start
  self.bus.log('Set handler for console events.', level=40)
stop
  self.bus.log('Removed handler for console events.', level=40)


**What is the expected behavior?**
It should report as INFO
 [cherrypy.error] [INFO] [18/Dec/2018:19:40:27] ENGINE Set handler for console events.


**What is the motivation / use case for changing the behavior?**
Looks like a bug as the log shows error though everything is good.


**Please tell us about your environment:**
It occurs in all versions. Tested in CherryPy 7.1.0.
OS: Windows

**Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, e.g. stackoverflow, gitter, etc.)

--- a/cherrypy/process/win32.py
+++ b/cherrypy/process/win32.py
@@ -28,7 +28,7 @@ class ConsoleCtrlHandler(plugins.SimplePlugin):
             self.bus.log('Could not SetConsoleCtrlHandler (error %r)' %
                          win32api.GetLastError(), level=40)
         else:
-            self.bus.log('Set handler for console events.', level=40)
+            self.bus.log('Set handler for console events.', level=20)
             self.is_set = True
 
     def stop(self):
@@ -46,7 +46,7 @@ class ConsoleCtrlHandler(plugins.SimplePlugin):
             self.bus.log('Could not remove SetConsoleCtrlHandler (error %r)' %
                          win32api.GetLastError(), level=40)
         else:
-            self.bus.log('Removed handler for console events.', level=40)
+            self.bus.log('Removed handler for console events.', level=20)
             self.is_set = False
 
     def handle(self, event):

